jsapp/scss/libs/_mdl.scss

```diff
@@ -1,7 +1,9 @@
 @import "../colors";
-
 // Theming variables, mixins and includes
 
+$black: #414453;
+$white: #fff;
```

Comment:
You don't need these either - use `$kobo-black` and `$kobo-white` when needed :)